.ci/generate_bear_requirements.py

```diff
@@ -155,18 +206,45 @@ def get_pip_requirements(requirements):
     pip_reqs, npm_reqs, gem_reqs = (
         get_all_requirements(get_all_bears(bear_dirs)))
 
-    requirements = {}
+    requirements = CommentedMap()
+    requirements.yaml_set_start_comment(
+                    "This is an automatically generated file.\n "
```

Comment:
indent four spaces only and use single quotes.

and rebase again ;-)

Reply:
Done!